pkg/metastore/test.go

```diff
@@ -0,0 +1,23 @@
+package metastore
+
+import (
+	"github.com/go-kit/log"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/otel/trace"
+
+	pb "github.com/parca-dev/parca/gen/proto/go/parca/metastore/v1alpha1"
+)
+
+func NewTestMetastore(
+	t require.TestingT,
+	logger log.Logger,
+	reg prometheus.Registerer,
+	tracer trace.Tracer,
+) pb.MetastoreServiceServer {
```

Comment:
I think it is good practice to call `t.Helper()` in these kind of functions, it outputs the line of the test if it fails instead of the helper

https://pkg.go.dev/testing#T.Helper